dist/Data-Dumper/Dumper.pm

```diff
@@ -1416,8 +1416,9 @@ distribution for more examples.)
 =head1 BUGS
 
 Due to limitations of Perl subroutine call semantics, you cannot pass an
-array or hash.  Prepend it with a C<\> to pass its reference instead.  This
-will be remedied in time, now that Perl has subroutine prototypes.
+array or hash.  Prepend it with a C<\> to pass its reference instead:
+Dumper C<\>(%a, @b, $c), C<\>%e, C<\>@f, C<\>$g.
```

Comment:
`\( ... )` is a list of references to each of the items in the parens. `\( @things )` is not a reference to `@things`, nor a reference to an anon-array with the items from `@things`:

```
% perl -MData::Dumper -E '
  say Dumper \( @ARGV ) 
'  -- two items
$VAR1 = \'two';
$VAR2 = \'items';
```

`\( %things )` does the same with the key/value pairs from the hash:

```
 % perl  -MData::Dumper -E '
  my %arg = @ARGV; 
  say Dumper \( %arg ) 
' -- one pair
$VAR1 = \'one';
$VAR2 = \'pair';
```

`Dumper \(%a, @b, $c)` is not a good example of how to avoid list flattening.